src/OpenTracing/Tag/Tags.cs

```diff
@@ -0,0 +1,139 @@
+        ///     HttpUrl records the url of the incoming request.
+        /// </summary>
+        public static readonly StringTag HttpUrl = new StringTag("http.url");
+
+        /// <summary>
+        ///     HttpStatus records the http status code of the response.
+        /// </summary>
+        public static readonly IntTag HttpStatus = new IntTag("http.status_code");
+
+        /// <summary>
+        ///     HttpMethod records the http method. Case-insensitive.
+        /// </summary>
+        public static readonly StringTag HttpMethod = new StringTag("http.method");
+
+        /// <summary>
+        ///     PeerHostIpv4 records IPv4 host address of the peer.
+        /// </summary>
+        public static readonly IntOrStringTag PeerHostIpv4 = new IntOrStringTag("peer.ipv4");
+
+        /// <summary>
+        ///     PeerHostIpv6 records the IPv6 host address of the peer.
+        /// </summary>
+        public static readonly StringTag PeerHostIpv6 = new StringTag("peer.ipv6");
+
+        /// <summary>
+        ///     PeerService records the service name of the peer.
+        /// </summary>
+        public static readonly StringTag PeerService = new StringTag("peer.service");
+
+        /// <summary>
+        ///     PeerHostname records the host name of the peer.
+        /// </summary>
+        public static readonly StringTag PeerHostname = new StringTag("peer.hostname");
+
+        /// <summary>
+        ///     PeerPort records the port number of the peer.
+        /// </summary>
+        public static readonly IntTag PeerPort = new IntTag("peer.port");
+
+        /// <summary>
+        ///     SamplingPriority determines the priority of sampling this Span.
+        /// </summary>
+        public static readonly IntTag SamplingPriority = new IntTag("sampling.priority");
+
+        /// <summary>
+        ///     SpanKind hints at the relationship between spans, e.g. client/server.
+        /// </summary>
+        public static readonly StringTag SpanKind = new StringTag("span.kind");
+
+        /// <summary>
+        ///     Component is a low-cardinality identifier of the module, library, or package that is instrumented.
+        /// </summary>
+        public static readonly StringTag Component = new StringTag("component");
+
+        /// <summary>
+        ///     Error indicates whether a Span ended in an error state.
+        /// </summary>
+        public static readonly BooleanTag Error = new BooleanTag("error");
+
+        /// <summary>
+        ///     DbType indicates the type of Database.
+        ///     For any SQL database, "sql". For others, the lower-case database category, e.g. "cassandra", "hbase", or "redis"
+        /// </summary>
+        public static readonly StringTag DbType = new StringTag("db.type");
+
+        /// <summary>
+        ///     DbInstance indicates the instance name of Database.
+        ///     If the jdbc.url="jdbc:mysql://127.0.0.1:3306/customers", instance name is "customers".
```

Comment:
there's no jdbc in .NET

Reply:
I've seen that there's also comments with JDBC in the current code so not a big issue - we can change this later.

Reply:
updated text to match the semantic_conventions spec. Planned to do this across the board after the port from java before

Reply:
resolved